cmake/std/atdm/utils/set_build_options.sh

```diff
@@ -111,6 +111,8 @@ elif [[ $ATDM_CONFIG_BUILD_NAME == *"gnu-6.1.0"* ]]; then
   export ATDM_CONFIG_COMPILER=GNU-6.1.0
 elif [[ $ATDM_CONFIG_BUILD_NAME == *"gnu-7.2.0"* ]]; then
   export ATDM_CONFIG_COMPILER=GNU-7.2.0
+elif [[ $ATDM_CONFIG_BUILD_NAME == *"gnu-7.3.1"* ]]; then
+  export ATDM_CONFIG_COMPILER=GNU-7.3.1
```

Comment:
FYI: If you have a custom_builds.sh file, then this never gets exectued.  Therefore, I would take this out.